css/index.css

```diff
@@ -104,13 +178,54 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.box:hover {
+    -webkit-animation:spin 1.5s linear infinite;
+    -moz-animation:spin 1.5s linear infinite;
+    animation:spin 1.5s linear infinite;
+}
+
+/*James box Colors */
+
+.box:nth-child(1) {
+    background: teal;
+  }
+  .box:nth-child(2) {
+    background: gold;
+  }
+  .box:nth-child(3) {
+    background: cadetblue;
+  }
+  .box:nth-child(4) {
+    background: coral;
+  }
+  .box:nth-child(5) {
+    background: crimson;
+  }
+  .box:nth-child(6) {
+    background: forestgreen;
+  }
+  .box:nth-child(7) {
+    background: darkorchid;
+  }
+  .box:nth-child(8) {
+    background: hotpink;
+  }
+  .box:nth-child(9) {
+    background: indigo;
+  }
+  .box:nth-child(10) {
+    background: dodgerblue;
+  }
```

Comment:
Another option instead of using ```:nth-child``` would have been to assign a second class to the individual boxes